webpack.config.js

```diff
@@ -113,6 +113,9 @@ const wordpressExternals = ( context, request, callback ) =>
  */
 // eslint-disable-next-line no-unused-vars
 function getWebpackConfig( { externalizeWordPressPackages = false } = {}, argv ) {
+	const cssFilename =
```

Comment:
Can we make this configurable? That would help SDK because if I now run:
```bash
npm run sdk:gutenberg -- --editor-script=client/gutenberg/extensions/todo/block.js
```

That would compile a file `client/gutenberg/extensions/todo/build/style-todo-editor.css` (with `style-`) instead of `client/gutenberg/extensions/todo/build/todo-editor.css`

Perhaps something like this:
```
function getWebpackConfig( { cssFilename, externalizeWordPressPackages = false } = {}, argv ) {
	if ( ! cssFilename ) {
		cssFilename = isDevelopment || calypsoEnv === 'desktop' ? 'style-[name].css' : 'style-[name].[chunkhash].css';
	}
```

And [in SDK](https://github.com/Automattic/wp-calypso/blob/a20f92fcfdba004e73ab725867b25733b7578de3/bin/sdk/gutenberg.js#L11):
```
const baseConfig = getBaseConfig( { cssFilename: '[name].css', externalizeWordPressPackages: true } );
```